README.md

```diff
@@ -15,7 +15,7 @@ Hello and welcome to the GitHub repository for The Coding Train website! This ho
 
 # Live Streams
 
-Live streams are announced as events on YouTube, [please subscribe to receive notifications for events](https://www.youtube.com/channel/UCvjgXvBlbQiydffZU7m1_aw/subscribe)!
+Live streams are announced as events on YouTube, [please subscribe to receive notifications for events](https://www.youtube.com/channel/UCvjgXvBlbQiydffZU7m1_aw/subscribe).
```

Comment:
I don't personally think this change improves things. Could you explain why you do?